runtime/src/snapshot_utils.rs

```diff
@@ -1594,6 +1594,7 @@ pub fn snapshot_bank(
     hash_for_testing: Option<Hash>,
     snapshot_type: Option<SnapshotType>,
 ) -> Result<()> {
+    let mut measure_snapshot_storages = Measure::start("snapshot-storages");
```

Comment:
I think I'd rather see a separate function:
maybe
get_snapshot_storages(root_bank, snapshot_type);
This is lots of lines to ultimately call `root_bank.get_snapshot_storages`

Reply:
Great idea. Done!